style.css

```diff
@@ -0,0 +1,54 @@
+/* @media only screen and (max-width: 600px) {
+    .main-nav {
+        font-size: 0.5em;
+        padding: 0;
+    }
+} */
+body{
+  margin: 0 auto;
+}
+
+.main-nav{
+  padding: 10px 10px 10px 10px;
+  background-color: aqua;
+  width: 100%;
+  text-align: right;
+}
+
+a{
+  font-size: 2em;
+  padding: 15px;
+  text-decoration: none;
+}
+
+@media (max-width: 780px){
+  img {
+      width: 300px;
+      height: 300px;
+  }
+}
```

Comment:
I think it is one of the best practices to put @media queries at the end of the stylesheet.

Reply:
- [ ] Polite comment. 👍🏾